drivers/spi/spi_ll_stm32.c

```diff
@@ -938,6 +938,15 @@ static void spi_stm32_irq_config_func_##id(const struct device *dev)		\
 #define STM32_SPI_USE_SUBGHZSPI_NSS_CONFIG(id)
 #endif
 
+#if !defined(CONFIG_SOC_SERIES_STM32F1X) \
+	&& (!defined(CONFIG_SOC_SERIES_STM32L1X) || defined(SPI_CR2_FRF))
+#define STM32_SPI_USE_FRAME_FORMAT(id) (DT_INST_PROP_OR(id, frame_format, \
+	SPI_FRAME_FORMAT_MOTOROLA) == SPI_FRAME_FORMAT_MOTOROLA) \
+	? LL_SPI_PROTOCOL_MOTOROLA : LL_SPI_PROTOCOL_TI
+#else
+#error "SPI Frame Format is not supported"
```

Comment:
```suggestion
#error "SPI Frame Format (TI) is not supported"
```

Reply:
some stm32L15x (like cat1 or cat 2)  only have SPI Motorola mode
some others like stm32F1xx have No frame format nor TI neither Motorola)

Reply:
> some others like stm32F1xx have No frame format nor TI neither Motorola)

Well then it is an issue as SPI API assumes you have to support one or the other.
cf https://github.com/zephyrproject-rtos/zephyr/pull/39990